src/webviews/apps/plus/graph/graph.tsx

```diff
@@ -538,7 +548,20 @@ export class GraphApp extends App<State> {
 				'--hover-row-border': isHighContrastTheme
 					? `1px dashed ${computedStyle.getPropertyValue('--color-graph-contrast-border')}`
 					: 'none',
-
+				'--scroll-marker-current': computedStyle.getPropertyValue('--color-graph-scroll-marker-current'),
```

Comment:
Can we avoid extra variables for these new ones? Meaning use 1 set of these new colors for the graph

Reply:
If we do need them, could these just be put them in `graph.scss` directly? Then we can skip the computing in JS.

```
body {
  --scroll-marker-current: var(--color-graph-scroll-marker-current);
}

```

Reply:
> Can we avoid extra variables for these new ones? Meaning use 1 set of these new colors for the graph

I'm having trouble understanding what you mean here - is the intent to use the existing css variables for colors already in the graph, and just pick ones out from those that would match, instead of sending in new ones here?

Reply:
I just want to avoid the situation we are in now where colors that GL exposes for themeability (Graph lanes, etc) which automatically get output into the DOM as CSS variables (by vscode) that we then read in js and set some other (differently named) CSS variable and then again for the Graph component itself. 

We should probably keep some insulation directly from the vscode colors directly (certainly for the graph) but @d13's solution to just set them in CSS is a great one for things we don't need to manipulate

Reply:
Sounds good, thanks both!

Reply:
Done in commit: [Remove redundant css properties](https://github.com/gitkraken/vscode-gitlens/pull/2489/commits/db0df4987a4a4a3dbcf694ca93569a1fe0ed5572)